features/access_management.md

```diff
@@ -0,0 +1,24 @@
+# Access management
+TokenD is based on a permissioned blockchain that limit the parties (it can be wether another person or external service) who can transact on the blockchain and that set who can serve the network by writing new blocks into the chain.

```

Comment:
either

Reply:
who can read and mutate the ledger.

Reply:
and that set who can serve the network by writing new blocks into the chain. - remove